api/app/src/external/commonwell/document/document-query-sandbox.ts

```diff
@@ -49,6 +61,8 @@ export async function sandboxGetDocRefsAndUpsert({
         s3BucketName: entry.s3Info.bucket,
       });
 
+      if (!conversionRequested) convertibleDocCount = convertibleDocCount - 1;
```

Comment:
If this fails wont the count be off https://github.com/metriport/metriport/blob/fb59202d6c677af8d0e0af117e3e4d78aee0fdfd/api/app/src/external/fhir-converter/converter.ts#L79-L97

Reply:
True story, good catch! We should add it on [the regular flow](https://vscode.dev/github/metriport/metriport/blob/103c03c876508bd1422813ac965cfc623f603fab/api/app/src/external/commonwell/document/document-query.ts#L482) as well.

Can you add to this PR, please, since the other fixes have been released to prod?

Reply:
Yup will do